kratos/linear_solvers/linear_solver.h

```diff
@@ -260,7 +260,7 @@ class LinearSolver
      */
     virtual void SetTolerance(double NewTolerance)
     {
-        KRATOS_WARNING("LinearSolver") << "WARNING: Accessed base function Kratos::LinearSolver::SetTolerance(double). This does nothing !" << std::endl;
+        KRATOS_WARNING("LinearSolver") << "Accessed base function Kratos::LinearSolver::SetTolerance(double). This does nothing !" << std::endl;
```

Comment:
```suggestion
        KRATOS_WARNING("LinearSolver") << "Accessed base function \"SetTolerance\". This does nothing !" << std::endl;
```

The location is already printed in the label